### PR TITLE
Resolve user paths relative to .boostraprc

### PR DESCRIPTION
**Pull request by danielfalk**
I got an "File to import not found or unreadable" error while trying to use bootstrap-loader because the working directory of the node process was not the same as the directory that the .bootstraprc file was in.  This pull request ensures that the paths are determined relative to the .bootstraprc file no matter what.

[Review on Reviewable](https://reviewable.io/reviews/shakacode/bootstrap-loader/22)

### CODE REVIEW

File: src/bootstrap.config.js [hunk 1]
```diff
@@ -63,15 +63,16 @@ export function createConfig({
   extractStyles,
 }) {
   if (isUserConfig) {
+    var configDir = path.dirname(userConfigPath);
```
Comment (justin808) [inline src/bootstrap.config.js]: const ?

**Comment by justin808** [comment]
@alexfedoseev Check this out.

@danielfalk Can you provide tests for this? Thanks so much!

**Comment by alexfedoseev** [comment]
@danielfalk Thanks! I'll check this out this week (most likely on weekend), in one scope w/ all other issues/PRs.

/cc: @justin808

File: src/bootstrap.config.js (hunk 1 above)
Comment (danielfalk) [inline src/bootstrap.config.js]: Thanks.  Updated.

**Comment by danielfalk** [comment]
I updated the pull request to use const instead of var.

@justin808, I don't see a test framework here.  Do you just mean how to reproduce?  For that, I would say you could put a `process.chdir(<some other path>)` in, say, `webpack.dev.config.js`.

**Comment by justin808** [comment]
@danielfalk Please add an example to the "examples" directory that shows this in action.

File: package.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "bootstrap-loader",
-  "version": "1.0.5",
+  "version": "1.0.5-isqft.2",
```
Comment (alexfedoseev) [inline package.json]: ?

**Comment by alexfedoseev** [comment]
@danielfalk It's in master. I transfered updates to npm scripts and `bootstrap.config.js` manually b/c of strange changes in `package.json`. Thanks and closing this.

**Pull request closed without merging**